docs/reference/extensions/django-framework.rst

```diff
@@ -29,6 +32,13 @@ There are 2 requirements to be able to use the ``django-framework`` extension:
    ``./<Rock name with - replaced by _>/<Rock name with - replaced by _>/manage.py``
    relative to the ``rockcraft.yaml`` file.
 
+For the project to make use of asynchronous Gunicorn workers:
+
+- The ``requirements.txt`` file must include ``gevent`` as a dependency.
+
+Read more
+:ref:`Gunicorn worker selection <django-gunicorn-worker-selection>`.
```

Comment:
On second look, let's cut these lines, too. _Requirements_ isn't the right place to link to another section in the doc. I think the reader can find their way to it on their own.
```suggestion
```